### PR TITLE
auth: Update executor auth tests

### PR DESCRIPTION
**Pull request by efritz**
c43eb41b9973128477def452910a9b9ecdd2b09b changed the way auth is accepted in the executors. This change in code revealed a test that never set a valid token for executors to begin with.

### CODE REVIEW

File: dev/authtest/code_intel_test.go [hunk 1]
```diff
@@ -113,7 +114,25 @@ func TestCodeIntelEndpoints(t *testing.T) {
...
+		// Update site configuration to remove any executor access token.
+		defer setExecutorAccessToken(t, "")()
+
+		resp, err := userClient.Get(*baseURL + "/.executors/")
+		if err != nil {
+			t.Fatal(err)
+		}
+		defer func() { _ = resp.Body.Close() }()
+
+		if resp.StatusCode/100 != 5 {
```
Comment (eseliger) [inline dev/authtest/code_intel_test.go:127]: 😆

File: dev/authtest/code_intel_test.go [hunk 2]
```diff
@@ -113,7 +114,25 @@ func TestCodeIntelEndpoints(t *testing.T) {
...
+			t.Fatalf(`Want status code 5xx error but got %d`, resp.StatusCode)
+		}
+	})
+
+	t.Run("executor endpoints (no access token supplied)", func(t *testing.T) {
```
Comment (eseliger) [inline dev/authtest/code_intel_test.go:132]:
```suggestion
	t.Run("executor endpoints (access token supplied)", func(t *testing.T) {
```
Or am I stupid?

**Review by eseliger — APPROVED** [review]

File: dev/authtest/code_intel_test.go (hunk 1 above)
Comment (bobheadxi) [inline dev/authtest/code_intel_test.go:127]: aside: shouldn't this look for a more specific status code? for example, in the failed run it looks like you return 500, although...

image

non-blocking though, let's get tests passing first 😁

**Review by bobheadxi — APPROVED** [review]
thank you for looking into this! LGTM on paper

File: dev/authtest/code_intel_test.go (hunk 2 above)
Comment (bobheadxi) [reply dev/authtest/code_intel_test.go:132]: I think this is "access token _set_ but not _supplied_"

Comment (efritz) [reply dev/authtest/code_intel_test.go:132]: No! Access token supplied!

File: dev/authtest/code_intel_test.go (hunk 1 above)
Comment (efritz) [reply dev/authtest/code_intel_test.go:127]: Hey that was *there*.

File: dev/authtest/code_intel_test.go (hunk 2 above)
Comment (eseliger) [inline dev/authtest/code_intel_test.go:132]:
```suggestion
	t.Run("executor endpoints (No! Access token supplied!)", func(t *testing.T) {
```

Comment (efritz) [reply dev/authtest/code_intel_test.go]: Updated.

File: dev/authtest/code_intel_test.go (hunk 1 above)
Comment (efritz) [reply dev/authtest/code_intel_test.go]: Updated.

File: dev/authtest/code_intel_test.go (hunk 2 above)
Comment (efritz) [reply dev/authtest/code_intel_test.go]: 🗡️

**Pull request merged**